app/models/service_provider.rb

```diff
@@ -19,7 +19,7 @@ class ServiceProvider < ApplicationRecord
   validate :certs_are_pems
 
   enum block_encryption: { 'none' => 0, 'aes256-cbc' => 1 }, _suffix: 'encryption'
-  enum identity_protocol: { openid_connect: 0, saml: 1 }
```

Comment:
There's a reference to `openid_connect` here:

https://github.com/18F/identity-dashboard/blob/889a03b6ee324c271dd30f120f118380bae11dc3/app/views/service_providers/show.html.erb#L92

Do we need to update it?

Do we have to worry about migrating any existing data with this change?

Reply:
All the good questions! Migrating data is not necessary since the database is just storing integer representations (although I should check if we were using any of the helper methods, e.g. `ServiceProvider#openid_connect?` - don't think so assuming our test coverage is _decent_).

That said, yes, that logic will break so I will fix it. Thanks for the close look!

Reply:
Oh, there are a few existing SPs that we'd probably want to move over (at least the ones in prod already) - I can find them and make the update once this is deployed.